### DISCUSSION TITLE
golang driver todo list

### DISCUSSION THREAD

**Discussion by fmoor** [geldata/gel#1912] [General]
These aspects of edgedb-go need to be finalized. 

- [ ] [serialization approach](https://github.com/edgedb/edgedb/discussions/1915)
- [ ] [query API](https://github.com/edgedb/edgedb/discussions/1917)
- [ ] transaction API
- [ ] error API
- [ ] internal SCRAM implementation?

**Comment by 1st1** [comment]
I'd prioritize `serialization approach` and `query API`. Can you start discussions on these two? The rest feels unimportant at this stage.

**Reply by fmoor** [reply]
discussions started :heavy_check_mark: see links in updated discussion :arrow_up:
